Cargo.toml

```diff
@@ -32,3 +32,6 @@ version = "0.3.0"
 git = "https://github.com/internet2-wg/rust-cyphernet"
 rev = "dee03a95abe4c964e5d9f8532c7dc76998dfeea7"
 version = "0.1.0"
+
+[patch.crates-io.radicle-git-ext]
+git = "https://github.com/radicle-dev/radicle-git"
```

Comment:
We should be using the crates.io version of this, maybe ask fintan to publish a new version if it's out of date.

Reply:
one of the packages in `radicle-git` (I think surf) depends on its local `radicle-git-ext` and so if we use crates.io version, we might get out of sync. 

either both has to be from crates.io or we pull them from git.

Reply:
We're almost ready to publish `radicle-surf` to crates again. I think once we've merged the current open PRs then we can release `0.10.0` or whatever it is